### ISSUE TITLE
Istiod crashes creating sidecar injector

### ISSUE THREAD

**Issue by PeteMac88** [istio/istio#43555]
### Bug Description

After the Kubernetes update to 1.22 the istiod pod crashes because it cannot initialize the sidecar injector. Se following the complete crash log of the pod. Can anyone help here because, in my staging environment, everything works as expected with the same istio and k8s version.

INFO: I had to cut the error log because of max length of the body

### Version

```prose
Istio:
client version: 1.9.3
control plane version: 1.15.2
data plane version: 1.15.2 (74 proxies)

Kubernetes version:
Client Version: v1.19.6-eks-49a6c0
Server Version: v1.22.17-eks-48e63af
```

### Additional Information

```
2023-02-23T09:44:05.268802Z	info	FLAG: --caCertFile=""
2023-02-23T09:44:05.268840Z	info	FLAG: --clusterAliases="[]"
2023-02-23T09:44:05.268845Z	info	FLAG: --clusterID="Kubernetes"
2023-02-23T09:44:05.268849Z	info	FLAG: --clusterRegistriesNamespace="istio-system"
2023-02-23T09:44:05.268852Z	info	FLAG: --configDir=""
2023-02-23T09:44:05.268856Z	info	FLAG: --ctrlz_address="localhost"
2023-02-23T09:44:05.268861Z	info	FLAG: --ctrlz_port="9876"
2023-02-23T09:44:05.268865Z	info	FLAG: --domain="cluster.local"
2023-02-23T09:44:05.268869Z	info	FLAG: --grpcAddr=":15010"
2023-02-23T09:44:05.268874Z	info	FLAG: --help="false"
2023-02-23T09:44:05.268878Z	info	FLAG: --httpAddr=":8080"
2023-02-23T09:44:05.268881Z	info	FLAG: --httpsAddr=":15017"
2023-02-23T09:44:05.268887Z	info	FLAG: --keepaliveInterval="30s"
2023-02-23T09:44:05.268892Z	info	FLAG: --keepaliveMaxServerConnectionAge="30m0s"
2023-02-23T09:44:05.268899Z	info	FLAG: --keepaliveTimeout="10s"
2023-02-23T09:44:05.268903Z	info	FLAG: --kubeconfig=""
2023-02-23T09:44:05.268908Z	info	FLAG: --kubernetesApiBurst="160"
2023-02-23T09:44:05.268915Z	info	FLAG: --kubernetesApiQPS="80"
2023-02-23T09:44:05.268919Z	info	FLAG: --log_as_json="false"
2023-02-23T09:44:05.268923Z	info	FLAG: --log_caller=""
2023-02-23T09:44:05.268926Z	info	FLAG: --log_output_level="default:info"
2023-02-23T09:44:05.268929Z	info	FLAG: --log_rotate=""
2023-02-23T09:44:05.268933Z	info	FLAG: --log_rotate_max_age="30"
2023-02-23T09:44:05.268937Z	info	FLAG: --log_rotate_max_backups="1000"
2023-02-23T09:44:05.268942Z	info	FLAG: --log_rotate_max_size="104857600"
2023-02-23T09:44:05.268946Z	info	FLAG: --log_stacktrace_level="default:none"
2023-02-23T09:44:05.268960Z	info	FLAG: --log_target="[stdout]"
2023-02-23T09:44:05.268968Z	info	FLAG: --meshConfig="./etc/istio/config/mesh"
2023-02-23T09:44:05.268972Z	info	FLAG: --monitoringAddr=":15014"
2023-02-23T09:44:05.268975Z	info	FLAG: --namespace="istio-system"
2023-02-23T09:44:05.268979Z	info	FLAG: --networksConfig="./etc/istio/config/meshNetworks"
2023-02-23T09:44:05.268982Z	info	FLAG: --profile="true"
2023-02-23T09:44:05.268994Z	info	FLAG: --registries="[Kubernetes]"
2023-02-23T09:44:05.268998Z	info	FLAG: --secureGRPCAddr=":15012"
2023-02-23T09:44:05.269001Z	info	FLAG: --shutdownDuration="10s"
2023-02-23T09:44:05.269008Z	info	FLAG: --tls-cipher-suites="[]"
2023-02-23T09:44:05.269012Z	info	FLAG: --tlsCertFile=""
2023-02-23T09:44:05.269017Z	info	FLAG: --tlsKeyFile=""
2023-02-23T09:44:05.269022Z	info	FLAG: --vklog="0"
2023-02-23T09:44:05.291761Z	info	klog	Config not found: /var/run/secrets/remote/config
2023-02-23T09:44:05.306835Z	info	initializing mesh configuration ./etc/istio/config/mesh
2023-02-23T09:44:05.322992Z	info	controllers	starting	controller=configmap istio
2023-02-23T09:44:05.323340Z	info	Loaded MeshNetworks config from Kubernetes API server.
2023-02-23T09:44:05.323358Z	info	mesh networks configuration updated to: {

}
2023-02-23T09:44:05.325774Z	info	Loaded MeshConfig config from Kubernetes API server.
2023-02-23T09:44:05.326430Z	info	mesh configuration updated to: {
    "proxyListenPort": 15001,
    "connectTimeout": "10s",
    "protocolDetectionTimeout": "0s",
    "ingressClass": "istio",
    "ingressService": "istio-ingressgateway",
    "ingressControllerMode": "STRICT",
    "enableTracing": true,
    "accessLogFile": "/dev/stdout",
    "defaultConfig": {
        "configPath": "./etc/istio/proxy",
        "binaryPath": "/usr/local/bin/envoy",
        "serviceCluster": "istio-proxy",
        "drainDuration": "45s",
        "parentShutdownDuration": "60s",
        "discoveryAddress": "istiod.istio-system.svc:15012",
        "proxyAdminPort": 15000,
        "controlPlaneAuthPolicy": "MUTUAL_TLS",
        "statNameLength": 189,
        "concurrency": 2,
        "tracing": {
            "zipkin": {
                "address": "zipkin.istio-system:9411"
            }
        },
        "statusPort": 15020,
        "terminationDrainDuration": "5s"
    },
    "outboundTrafficPolicy": {
        "mode": "ALLOW_ANY"
    },
    "enableAutoMtls": true,
    "trustDomain": "cluster.local",
    "defaultServiceExportTo": [
        "*"
    ],
    "defaultVirtualServiceExportTo": [
        "*"
    ],
    "defaultDestinationRuleExportTo": [
        "*"
    ],
    "rootNamespace": "istio-system",
    "localityLbSetting": {
        "enabled": true
    },
    "dnsRefreshRate": "60s",
    "thriftConfig": {

    },
    "enablePrometheusMerge": true,
    "extensionProviders": [
        {
            "name": "prometheus",
            "prometheus": {

            }
        },
        {
            "name": "stackdriver",
            "stackdriver": {

            }
        },
        {
            "name": "envoy",
            "envoyFileAccessLog": {
                "path": "/dev/stdout"
            }
        }
    ],
    "defaultProviders": {

    }
}
2023-02-23T09:44:05.338984Z	info	initializing mesh networks from mesh config watcher
2023-02-23T09:44:05.339401Z	info	mesh configuration: {
    "proxyListenPort": 15001,
    "connectTimeout": "10s",
    "protocolDetectionTimeout": "0s",
    "ingressClass": "istio",
    "ingressService": "istio-ingressgateway",
    "ingressControllerMode": "STRICT",
    "enableTracing": true,
    "accessLogFile": "/dev/stdout",
    "defaultConfig": {
        "configPath": "./etc/istio/proxy",
        "binaryPath": "/usr/local/bin/envoy",
        "serviceCluster": "istio-proxy",
        "drainDuration": "45s",
        "parentShutdownDuration": "60s",
        "discoveryAddress": "istiod.istio-system.svc:15012",
        "proxyAdminPort": 15000,
        "controlPlaneAuthPolicy": "MUTUAL_TLS",
        "statNameLength": 189,
        "concurrency": 2,
        "tracing": {
            "zipkin": {
                "address": "zipkin.istio-system:9411"
            }
        },
        "statusPort": 15020,
        "terminationDrainDuration": "5s"
    },
    "outboundTrafficPolicy": {
        "mode": "ALLOW_ANY"
    },
    "enableAutoMtls": true,
    "trustDomain": "cluster.local",
    "defaultServiceExportTo": [
        "*"
    ],
    "defaultVirtualServiceExportTo": [
        "*"
    ],
    "defaultDestinationRuleExportTo": [
        "*"
    ],
    "rootNamespace": "istio-system",
    "localityLbSetting": {
        "enabled": true
    },
    "dnsRefreshRate": "60s",
    "thriftConfig": {

    },
    "enablePrometheusMerge": true,
    "extensionProviders": [
        {
            "name": "prometheus",
            "prometheus": {

            }
        },
        {
            "name": "stackdriver",
            "stackdriver": {

            }
        },
        {
            "name": "envoy",
            "envoyFileAccessLog": {
                "path": "/dev/stdout"
            }
        }
    ],
    "defaultProviders": {

    }
}
2023-02-23T09:44:05.339423Z	info	version: 1.15.2-b542583af7f8240ad0c9cab56eb9f8d55e1357ab-Clean
2023-02-23T09:44:05.339663Z	info	flags: {
   "ServerOptions": {
      "HTTPAddr": ":8080",
      "HTTPSAddr": ":15017",
      "GRPCAddr": ":15010",
      "MonitoringAddr": ":15014",
      "EnableProfiling": true,
      "TLSOptions": {
         "CaCertFile": "",
         "CertFile": "",
         "KeyFile": "",
         "TLSCipherSuites": null,
         "CipherSuits": null
      },
      "SecureGRPCAddr": ":15012"
   },
   "InjectionOptions": {
      "InjectionDirectory": "./var/lib/istio/inject"
   },
   "PodName": "istiod-67d4568846-2tdh5",
   "Namespace": "istio-system",
   "Revision": "default",
   "MeshConfigFile": "./etc/istio/config/mesh",
   "NetworksConfigFile": "./etc/istio/config/meshNetworks",
   "RegistryOptions": {
      "FileDir": "",
      "Registries": [
         "Kubernetes"
      ],
      "KubeOptions": {
         "SystemNamespace": "",
         "MeshServiceController": null,
         "DomainSuffix": "cluster.local",
         "ClusterID": "Kubernetes",
         "ClusterAliases": {},
         "Metrics": null,
         "XDSUpdater": null,
         "NetworksWatcher": null,
         "MeshWatcher": null,
         "EndpointMode": 1,
         "KubernetesAPIQPS": 80,
         "KubernetesAPIBurst": 160,
         "SyncTimeout": 0,
         "DiscoveryNamespacesFilter": null
      },
      "ClusterRegistriesNamespace": "istio-system",
      "KubeConfig": "",
      "DistributionCacheRetention": 60000000000,
      "DistributionTrackingEnabled": true
   },
   "CtrlZOptions": {
      "Port": 9876,
      "Address": "localhost"
   },
   "KeepaliveOptions": {
      "Time": 30000000000,
      "Timeout": 10000000000,
      "MaxServerConnectionAge": 1800000000000,
      "MaxServerConnectionAgeGrace": 10000000000
   },
   "ShutdownDuration": 10000000000,
   "JwtRule": ""
}
2023-02-23T09:44:05.339671Z	info	initializing mesh handlers
2023-02-23T09:44:05.339724Z	info	model	reloading network gateways
2023-02-23T09:44:05.339731Z	info	creating CA and initializing public key
2023-02-23T09:44:05.339745Z	info	Using istiod file format for signing ca files
2023-02-23T09:44:05.339755Z	info	Use self-signed certificate as the CA certificate
2023-02-23T09:44:05.345177Z	info	pkica	Load signing key and cert from existing secret istio-system:istio-ca-secret
2023-02-23T09:44:05.345580Z	info	pkica	Using existing public key: -----BEGIN CERTIFICATE-----
MIIC/TCCAeWgAwIBAgIRAJmAvKJzneEgCL5//teiRTQwDQYJKoZIhvcNAQELBQAw
GDEWMBQGA1UEChMNY2x1c3Rlci5sb2NhbDAeFw0yMTA0MzAxMTQ5NDRaFw0zMTA0
MjgxMTQ5NDRaMBgxFjAUBgNVBAoTDWNsdXN0ZXIubG9jYWwwggEiMA0GCSqGSIb3
DQEBAQUAA4IBDwAwggEKAoIBAQDajLYsGARHM00sGq/PUDrjSW2myvaeB0sf2c3J
2UcNp9YzdL8oyCMP9R3CCYW6tbl6l+qZhCywoYa+C9xAWaUUJfHfAToWC/bg5WOi
Q+z+Am5CkqrY6qF+WlQEFukHPCoyxDyLCzMXJO6tfyh052SNlvT4TLTgf1/7Wrmf
1vQrEENpc1dxBRpGnyCEsR9IJHVdr7nOsQAt3jRhoM6yXYnI4o9C/9UQ0bTnGQJu
JzEA2pMiARjwigb//8pUQTXvwksmarUbF1uS4ERVVohL4KLxW9WHVTxCzyN3dH2i
dqvA826TCzXYFNQSYbiWdzM0RO4DrI2e0AMX8y9OgeLvJhJZAgMBAAGjQjBAMA4G
A1UdDwEB/wQEAwICBDAPBgNVHRMBAf8EBTADAQH/MB0GA1UdDgQWBBTUNSw6NIxi
K/Gmv7OiVFHrHzUbajANBgkqhkiG9w0BAQsFAAOCAQEAuiE/X/74uMlQNUqtbyMh
+tfx5ixCddLEiPYE+cTA24BZDNvhRLhpZ31mNaViR3aP/gIk/esETvt2489KK5v2
aJ0cd3u0D6tfO6ZUSuH0zN9lFCOXA3W2pYMDGRZskzANYEgvyrm+Yadmy6hY9p/h
MKLCiGudSCFPyn5oCGp4MqwG/e7OPTnvgH41wvsWD7dvKj8Q3vPBIvoJOwoekIfy
cy+h85Eot7/Sv5+3LcRQYxK1OjTWaQyB6xBnYFMuk43afvDU9cUIJLd9A3JVo8ji
qKLXH0rpPhayhJjX5MekWmj46LylX4IW476RWAXJVfHL3jS5hNhf6IpJJ4xQ6mV/
WA==
-----END CERTIFICATE-----

2023-02-23T09:44:05.345609Z	info	rootcertrotator	Set up back off time 26m23s to start rotator.
2023-02-23T09:44:05.345639Z	info	initializing controllers
2023-02-23T09:44:05.345687Z	info	No certificates specified, skipping K8S DNS certificate controller
2023-02-23T09:44:05.345990Z	info	rootcertrotator	Jitter is enabled, wait 26m23s before starting root cert rotator.
2023-02-23T09:44:05.674420Z	warn	kube	Skipping CRD gateway.networking.k8s.io/v1alpha2/ReferenceGrant as it is not present
2023-02-23T09:44:05.674452Z	warn	kube	Skipping CRD gateway.networking.k8s.io/v1alpha2/ReferencePolicy as it is not present
2023-02-23T09:44:05.674457Z	warn	kube	Skipping CRD gateway.networking.k8s.io/v1alpha2/TCPRoute as it is not present
2023-02-23T09:44:05.674460Z	warn	kube	Skipping CRD gateway.networking.k8s.io/v1alpha2/TLSRoute as it is not present
2023-02-23T09:44:05.674464Z	warn	kube	Skipping CRD gateway.networking.k8s.io/v1beta1/GatewayClass as it is not present
2023-02-23T09:44:05.674467Z	warn	kube	Skipping CRD gateway.networking.k8s.io/v1beta1/Gateway as it is not present
2023-02-23T09:44:05.674471Z	warn	kube	Skipping CRD gateway.networking.k8s.io/v1beta1/HTTPRoute as it is not present
2023-02-23T09:44:05.674909Z	info	Adding Kubernetes registry adapter
2023-02-23T09:44:05.674949Z	info	initializing Istiod DNS certificates host: istiod.istio-system.svc, custom host: 
2023-02-23T09:44:05.764425Z	info	Generating istiod-signed cert for [istiod.istio-system.svc istiod-remote.istio-system.svc istio-pilot.istio-system.svc]:
 -----BEGIN CERTIFICATE-----
MIIDajCCAlKgAwIBAgIRAK9YhY/mJyCZDWICldODFU0wDQYJKoZIhvcNAQELBQAw
GDEWMBQGA1UEChMNY2x1c3Rlci5sb2NhbDAeFw0yMzAyMjMwOTQyMDVaFw0zMzAy
MjAwOTQ0MDVaMAAwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQCf63Wb
PwjpyoTOW4fHMDyMoqaeiaY77gX8UlflyGDWdH9sy4/0m4lumeKNG8VKH7bXiCGJ
e/nodfvLzq1boqZvBCQt86RVQ41ys7AUwt1gP5t31yz0Obuf7nBUsb752l/6LIkz
cRy+AbF1+4IG0l9XVFp70kOTvjJ6XAFydQF6jR1LXoteatPkeJGlKHcYyBe5iVk6
3XiezjcyqLYUCjoGCSSM4LdG5FTauCmCxwiQbp2LtfjntZ+4Z0U21XLX3vkpydhI
KDM5qmqoKPpbfeZRprMSqtrHR+zYbEjDCJVXw9JlRNc0kSSDP5G7gOG7vanLS2bu
NJNpRi2E7g40KEE/AgMBAAGjgcYwgcMwDgYDVR0PAQH/BAQDAgWgMB0GA1UdJQQW
MBQGCCsGAQUFBwMBBggrBgEFBQcDAjAMBgNVHRMBAf8EAjAAMB8GA1UdIwQYMBaA
FNQ1LDo0jGIr8aa/s6JUUesfNRtqMGMGA1UdEQEB/wRZMFeCF2lzdGlvZC5pc3Rp
by1zeXN0ZW0uc3Zjgh5pc3Rpb2QtcmVtb3RlLmlzdGlvLXN5c3RlbS5zdmOCHGlz
dGlvLXBpbG90LmlzdGlvLXN5c3RlbS5zdmMwDQYJKoZIhvcNAQELBQADggEBADwy
jRHSgeK+Xi++K4JK98e8zrnWMR1a61ExNneMYUzP92r2xCX9X43wL7g49vi6WUJB
s0SdsnifrCLIerCpLl+rci1MKu+3DUVGymbR1ZQUNOxTok+q3JRtxTCGnU4WnD+W
tgStJAm0FwQOpd6ZEEqbR1N3oYma1SrNKI+XfcrWiFFI46RhhGpYAy/ANtsG79z6
WcZzpD6bPkFv317ONh+OiM8J36TvLfZKw0cySXp4G+YLAGdNa7kyyDF2HT2mQzt9
qkDRWYtUthUiH3JZVHLI7/fCWih2yTuSMtvjHvYhDujQ5g8eUiECtmR1vBuXcflI
RsdP+2mV4F94Z81XICE=
-----END CERTIFICATE-----

2023-02-23T09:44:05.764493Z	info	Using istiod file format for signing ca files
2023-02-23T09:44:05.764507Z	info	No plugged-in cert at etc/cacerts/ca-key.pem; self-signed cert is used
2023-02-23T09:44:05.764676Z	info	x509 cert - Issuer: "O=cluster.local", Subject: "", SN: af58858fe62720990d620295d383154d, NotBefore: "2023-02-23T09:42:05Z", NotAfter: "2033-02-20T09:44:05Z"
2023-02-23T09:44:05.764689Z	info	Istiod certificates are reloaded
2023-02-23T09:44:05.764742Z	info	spiffe	Added 1 certs to trust domain cluster.local in peer cert verifier
2023-02-23T09:44:05.764753Z	info	initializing secure discovery service
2023-02-23T09:44:05.764970Z	info	initializing secure webhook server for istiod webhooks
2023-02-23T09:44:05.780479Z	info	initializing sidecar injector
2023-02-23T09:44:05.796889Z	info	Failed to parse template: template: inject:13: function "toJsonMap" not defined apiVersion: v1
kind: ServiceAccount
metadata:
  name: {{.ServiceAccount | quote}}
  namespace: {{.Namespace | quote}}
---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: {{.DeploymentName | quote}}
  namespace: {{.Namespace | quote}}
  annotations:
    {{- toJsonMap (omit .Annotations "kubectl.kubernetes.io/last-applied-configuration" "gateway.istio.io/name-override" "gateway.istio.io/service-account") | nindent 4 }}
  labels:
    {{- toJsonMap .Labels | nindent 4 }}
  ownerReferences:
  - apiVersion: gateway.networking.k8s.io/v1beta1
    kind: Gateway
    name: {{.Name}}
    uid: "{{.UID}}"
spec:
  selector:
    matchLabels:
      istio.io/gateway-name: {{.Name}}
  template:
    metadata:
      annotations:
        {{- toJsonMap
          (omit .Annotations "kubectl.kubernetes.io/last-applied-configuration" "gateway.istio.io/name-override" "gateway.istio.io/service-account")
          (strdict
            "prometheus.io/path" "/stats/prometheus"
            "prometheus.io/port" "15020"
            "prometheus.io/scrape" "true"
          ) | nindent 8 }}
      labels:
        {{- toJsonMap
          (strdict
            "sidecar.istio.io/inject" "false"
            "service.istio.io/canonical-name" .DeploymentName
            "service.istio.io/canonical-revision" "latest"
           )
          .Labels
          (strdict "istio.io/gateway-name" .Name) | nindent 8}}
    spec:
      {{- if .KubeVersion122 }}
      {{/* safe since 1.22: https://github.com/kubernetes/kubernetes/pull/103326. */}}
      securityContext:
        sysctls:
        - name: net.ipv4.ip_unprivileged_port_start
          value: "0"
      {{- end }}
      serviceAccountName: {{.ServiceAccount | quote}}
      containers:
      - name: istio-proxy
        image: "{{ .ProxyImage }}"
        {{with .Values.global.imagePullPolicy }}imagePullPolicy: "{{.}}"{{end}}
        securityContext:
        {{- if .KubeVersion122 }}
          # Safe since 1.22: https://github.com/kubernetes/kubernetes/pull/103326
          capabilities:
            drop:
            - ALL
          allowPrivilegeEscalation: false
          privileged: false
          readOnlyRootFilesystem: true
          runAsUser: 1337
          runAsGroup: 1337
          runAsNonRoot: true
        {{- else }}
          capabilities:
            drop:
            - ALL
            add:
            - NET_BIND_SERVICE
          runAsUser: 0
          runAsGroup: 1337
          runAsNonRoot: false
          allowPrivilegeEscalation: true
          readOnlyRootFilesystem: true
        {{- end }}
        ports:
        - containerPort: 15021
          name: status-port
          protocol: TCP
        - containerPort: 15090
          protocol: TCP
          name: http-envoy-prom
        args:
        - proxy
        - router
        - --domain
        - $(POD_NAMESPACE).svc.{{ .Values.global.proxy.clusterDomain }}
        - --proxyLogLevel
        - {{ annotation .ObjectMeta `sidecar.istio.io/logLevel` .Values.global.proxy.logLevel | quote}}
        - --proxyComponentLogLevel
        - {{ annotation .ObjectMeta `sidecar.istio.io/componentLogLevel` .Values.global.proxy.componentLogLevel | quote}}
        - --log_output_level
        - {{ annotation .ObjectMeta `sidecar.istio.io/agentLogLevel` .Values.global.logging.level | quote}}
      {{- if .Values.global.sts.servicePort }}
        - --stsPort={{ .Values.global.sts.servicePort }}
      {{- end }}
      {{- if .Values.global.logAsJson }}
        - --log_as_json
      {{- end }}
      {{- if .Values.global.proxy.lifecycle }}
        lifecycle:
          {{ toYaml .Values.global.proxy.lifecycle | indent 6 }}
      {{- end }}
        env:
        - name: JWT_POLICY
          value: {{ .Values.global.jwtPolicy }}
        - name: PILOT_CERT_PROVIDER
          value: {{ .Values.global.pilotCertProvider }}
        - name: CA_ADDR
        {{- if .Values.global.caAddress }}
          value: {{ .Values.global.caAddress }}
        {{- else }}
          value: istiod{{- if not (eq .Values.revision "") }}-{{ .Values.revision }}{{- end }}.{{ .Values.global.istioNamespace }}.svc:15012
        {{- end }}
        - name: POD_NAME
          valueFrom:
            fieldRef:
              fieldPath: metadata.name
        - name: POD_NAMESPACE
          valueFrom:
            fieldRef:
              fieldPath: metadata.namespace
        - name: INSTANCE_IP
          valueFrom:
            fieldRef:
              fieldPath: status.podIP
        - name: SERVICE_ACCOUNT
          valueFrom:
            fieldRef:
              fieldPath: spec.serviceAccountName
        - name: HOST_IP
          valueFrom:
            fieldRef:
              fieldPath: status.hostIP
        - name: PROXY_CONFIG
          value: |
                 {{ protoToJSON .ProxyConfig }}
        - name: ISTIO_META_POD_PORTS
          value: "[]"
        - name: ISTIO_META_APP_CONTAINERS
          value: ""
        - name: ISTIO_META_CLUSTER_ID
          value: "{{ valueOrDefault .Values.global.multiCluster.clusterName `Kubernetes` }}"
        - name: ISTIO_META_NODE_NAME
          valueFrom:
            fieldRef:
              fieldPath: spec.nodeName
        - name: ISTIO_META_INTERCEPTION_MODE
          value: "{{ .ProxyConfig.InterceptionMode.String }}"
  
```

**Comment by hzxuzhonghu** [comment]
@PeteMac88 Please paste the crash logs

**Comment by howardjohn** [comment]
You are installing the templates from `master` branch. This is NOT Istio 1.15.2,. just an Istio 1.15 image with `master` configmaps. This is not a support configuration. 

Not sure exactly how you got in this scenario, but your installation mechanism is not correct.

**Comment by blowfishpro** [comment]
We ran into this issue too.  We discovered that it was because we installed the operator via helm directly from git without overriding the values `hub` and `tag`.  It looks like in git these take the values `hub: gcr.io/istio-testing`, `tag: latest` (even if you reference a specific git tag), whereas in the packaged Istio releases they take values like `hub: docker.io/istio`, `tag: 1.17.1`.

It would be nice if the operator charts were published somewhere so that we could use the final versions in our GitOps workflow but we're also likely to move away from the operator to a pure helm installation in the near future as we're not really utilizing the capabilities of the operator.
